scripts/cumulus-v9.2.0/data_migration2.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# run this lambda after deploying the cumulus module
+
+PAYLOAD=$(echo '{"executionMigrationParams": { "parallelScanSegments": 50, "writeConcurrency": 50 }}')
```

Comment:
We got an error here that was fixed by piping the JSON to `base64`, as shown in the Cumulus docs - https://nasa.github.io/cumulus/docs/upgrade-notes/upgrade-rds#5-run-the-second-data-migration

Was the `base64` call omitted intentionally?

```suggestion
PAYLOAD=$(echo '{"executionMigrationParams": { "parallelScanSegments": 50, "writeConcurrency": 50 }}' | base64)
```

Reply:
hmm weird, when we ran this locally, the payload needed to be base64-encoded, when running through Bamboo, it seems to be expecting standard plaintext JSON 🤔

Reply:
There was some chatter in Slack #cumulus-internal about this.  If you are using awscli v1.x, no base64.  If using awscli v2.x, base64 is required.

Reply:
Are we ready to merge this.  I see @mckadesorensen signed off on it.  @michaeljb if you're ok with it, I'm fine with either one of you merging it in.

Reply:
@michaeljb @mckadesorensen would one of you mind merging this change in, unless you think it needs more testing.  I have used it to upgrade all my instances.  I think there might be some issues with the Cumulus migration code for some people, but not CIRRUS, am I right?  I'd like to go ahead and create a release so we'll be ready for the next version of Cumulus.

Reply:
Sorry, I didn't notice your last message. I have tested it too and it is working on my end.